### DISCUSSION TITLE
Find cause of graph dip

### DISCUSSION THREAD

**Discussion by cbirkenbeul** [ghostfolio/ghostfolio#3100] [General]
Hi,
are there any tips how I can find the root course of this graph issue? There were no trade on this day. Yesterday I updated all market date of all assets and currencies.

<img width="916" alt="Bildschirmfoto 2024-03-05 um 12 55 17" src="https://github.com/ghostfolio/ghostfolio/assets/1380162/48462dc9-b6e6-4a41-9d24-91f09a395368">

**Comment by pedrommvv** [comment]
Check for stock splits or spin offs in your holdings
